compose-files/docker-compose.yml

```diff
@@ -0,0 +1,98 @@
+version: '3'
+services:
+  indexd:
+    image: "quay.io/cdis/indexd"
+    command: "bash waitForPostgres.sh"
+    networks:
+      - devnet
+    volumes:
+      - ./apis_configs/indexd_settings.py:/var/www/indexd/local_settings.py
+      - ./apis_configs/creds.json:/var/www/indexd/creds.json
+      - ./apis_configs/config_helper.py:/var/www/indexd/config_helper.py
+      - ./waitForPostgres.sh:/var/www/indexd/waitForPostgres.sh
+    depends_on:
+      - postgres
+  fence:
+    image: "quay.io/cdis/fence"
+    networks:
+      - devnet
+    volumes:
+      - ./apis_configs/fence_settings.py:/var/www/fence/local_settings.py
+      - ./apis_configs/creds.json:/var/www/fence/creds.json
```

Comment:
Where do you extract the `indexd` block from `creds.json` ?  The `config_helper` and `settings.py` files expect the mounted `creds.json` to just have the block appropriate for the running service

Reply:
Oh my workaround for the `config_helper` and `settings.py` files expecting just the service-specific block was to change the code in the `settings.py` files to access `creds.json` by key. In the compose-services repo, the `settings.py` files are directly changed, but here in cloud automation I used sed to modify the lines